### PR TITLE
fix: Always pay attention to return val of api.SendRecords

### PR DESCRIPTION
**Pull request by obs-gh-irismcleary**

### CODE REVIEW

File: pkg/service/ec2/addresses.go [hunk 1]
```diff
@@ -40,6 +40,8 @@ func (fn *DescribeAddresses) New(name string, config interface{}) ([]api.Request
 		if err != nil {
 			return err
 		}
+
+		// TODO: not noticing return value, but does it matter?
```
Comment (obs-gh-irismcleary) [inline pkg/service/ec2/addresses.go]: @jta it returns right after this, but would it help to return an error instead? I'm having some trouble tracing how this actually gets called

Comment (jta) [reply pkg/service/ec2/addresses.go]: If you look at `api.SendRecords`, it returns `false` if `<-ctx.Done()`. So I think the correct behavior here would be to return `ctx.Err()` in that case.

Comment (obs-gh-irismcleary) [reply pkg/service/ec2/addresses.go]: cool, that's what I was thinking—thanks for validating

File: pkg/service/organizations/organization.go [hunk 2]
```diff
@@ -43,7 +43,11 @@ func (fn *DescribeOrganization) New(name string, config interface{}) ([]api.Requ
 		} else if err != nil {
 			return err
 		}
-		api.SendRecords(ctx, ch, name, &DescribeOrganizationOutput{output})
+
+		if err := api.SendRecords(ctx, ch, name, &DescribeOrganizationOutput{output}); err != nil {
```
Comment (obs-gh-irismcleary) [inline pkg/service/organizations/organization.go]: can just return api.SendRecords() here

**Review by jta — APPROVED** [review]

**Pull request merged**